mindmeld/augmentation.py

```diff
@@ -241,6 +262,167 @@ def __init__(self, batch_size, language, paths, path_suffix, resource_loader):
             "max_length": 60,
         }
 
+    def _download_model(self):
+        logger.info("Downloading paraphrase model from %s", PARAPHRASER_RETAIN_ENTITIES_URL)
+
+        # Make the folder that will contain the model folder
+        if not os.path.exists(EMBEDDINGS_FOLDER_PATH):
+            os.makedirs(EMBEDDINGS_FOLDER_PATH)
+
+        with TqdmUpTo(unit="B", unit_scale=True, miniters=1, desc='') as t:
+            try:
+                urlretrieve(PARAPHRASER_RETAIN_ENTITIES_URL,
+                            PARAPHRASER_FILE_PATH,
+                            reporthook=t.update_to)
+            except ConnectionError as e:
+                logger.error("Model download failed with error: %s", e)
+                return
+        try:
+            with zipfile.ZipFile(PARAPHRASER_FILE_PATH, 'r') as zip_ref:
+                zip_ref.extractall(EMBEDDINGS_FOLDER_PATH)
+        except zipfile.BadZipfile:
+            logger.error("Unable to extract zip file. Try downloading the model again.")
+        return
+
+    def _prepare_inputs(self, queries):
+        """Processes input as expected by the two different English models
+        Example:
+            The default model requires just <unannotated text>
+            The retain_entities model requires <unannotated text> <EOS> <entity values>
+
+        Args:
+            queries (list(str)): List of application queries.
+
+        Return:
+            model_inputs (list(str)): List of queries to be paraphrased in the
+                                      format required by the models
+            processed_entities (list(ProcessedQuery)): List of ProcessedQuery
+                                                       with entity annotations
+        """
+        model_inputs = []
+        processed_entities = []
+        for query in queries:
+            processed_query = load_query(query,
+                                         query_factory=self._resource_loader.query_factory)
+            if self.retain_entities:
+                text = [processed_query.query.text.strip().lower(), EOS_TOKEN]
+                for entity in processed_query.entities:
+                    text.append(entity.text.lower())
+                processed_entities.append(processed_query)
+                model_inputs.append(' '.join(text))
+            else:
+                model_inputs.append(processed_query.query.text.strip())
```

Comment:
Nit: Consider adding this line before the if-else block; 
`query_text = processed_query.query.text.strip()`

Then in the if block you can do:
`text = [query_text.lower(), EOS_TOKEN]`

And in the else block:
`model_inputs.append(query_text)`